docs/services/esi/pagination/cursor-based.md

```diff
@@ -0,0 +1,96 @@
+# Cursor-based Pagination
+
+Cursor-based pagination uses opaque tokens to navigate through datasets, unlike traditional offset-based pagination that uses page numbers.
```

Comment:
Could use an explanation on what "opaque" means in this context, especially that it's not that common of an English word.